src/desktop/apps/article/components/InfiniteScrollNewsArticle.js

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react'
+import _ from 'underscore'
+import PropTypes from 'prop-types'
+import Waypoint from 'react-waypoint'
+import styled from 'styled-components'
+import { positronql as _positronql } from 'desktop/lib/positronql'
+import { Article } from 'reaction/Components/Publishing'
+import { newsArticlesQuery } from 'desktop/apps/article/queries/articles'
+import { setupFollows, setupFollowButtons } from './FollowButton.js'
+
+// FIXME: Rewire
+let positronql = _positronql
```

Comment:
@eessex - can you go ahead and re-add `babel-plugin-rewire@1.2.0-rc.1` and scope it to a test environment in `.babelrc`? Babel 7 issues have been fixed following the above release so proceed as we did before. I'm going to sweep through and clean up these FIXME's after I finish sprint work

Reply:
Did this ever get added?